spec.bs

```diff
@@ -30,6 +30,7 @@ spec: ECMASCRIPT; urlPrefix: https://tc39.es/ecma262/
   type: dfn
     text: ParseScript; url: sec-parse-script
     text: abrupt completion; url: sec-completion-record-specification-type
+    text: normal completion; url: sec-completion-record-specification-type

```

Comment:
Use [=ECMAScript/normal completion=] to refer to ECMAScript terms without needing to make a custom anchor.

Reply:
I think you'r right, but the fact that this spec does not do this might be my fault from before I really learned how [to use Webdex](https://dontcallmedom.github.io/webdex/n.html#normal%20completion%40%40ECMAScript%40dfn)

Reply:
This was marked as resolved but I believe because we can reference `ECMAScript/normal completion`, and that's probably better, maybe we should do that in this PR (and clean up the others later).